### PR TITLE
Copy updates

### PR DESCRIPTION
**Pull request by amyyf**
- Updates README and adds deploy script
- Addresses various feedback for copy and wording
- Changes alerts to buttons (the 'info' color for buttons did not meet a11y standards 😢)
- Fixes some misc styling and spacing

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.App {
+  height: 150vh;
```
Comment (sfrieson) [inline src/components/App/App.css]:
```suggestion
  min-height: 150vh;
```

File: src/components/App/index.js [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
+import Button from 'react-bootstrap/Button';
 import Container from 'react-bootstrap/Container';
 import List from '../List';
-import Row from 'react-bootstrap/Row';
 import Toggle from '../Toggle';
 
 function App() {
   const formLink = 'https://docs.google.com/forms/d/1j_Cha4mhNiegs8mDWGcPGQczYlMxnzYiQw8uKAK7ujY/viewform?edit_requested=true#responses';
-  const responseLink = '/#';
+  const responseEmail = 'tktktk';
```
Comment (sfrieson) [inline src/components/App/index.js]: Adding a `// TODO` is easy to search for later.

Comment (sfrieson) [reply src/components/App/index.js]: Both of these might actually be better suited in a `config.json` and imported here.

File: src/components/App/index.js [hunk 3]
```diff
@@ -56,12 +50,13 @@ function App() {
...
       />
-      <AlertWrapper
-        link={responseLink}
-        linkText="Email us"
-        textAfterLink="to respond to a listing"
-        variant="info"
-      />
+      <Button
+        className="mb-3"
+        href={`mailto:${responseEmail}`}
```
Comment (sfrieson) [inline src/components/App/index.js]: Does the button component accept an `href`?

Comment (amyyf) [reply src/components/App/index.js]: adding a href prop will automatically render an <a /> element, pretty convenient

File: public/index.html [hunk 4]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>LMCC Community Assistance</title>
```
Comment (sfrieson) [inline public/index.html]: 💯

**Pull request merged**